src/commands/infra/generate.test.ts

```diff
@@ -0,0 +1,64 @@
+import fs, { chmod } from "fs";
+import * as os from "os";
+import path from "path";
+import {
+  disableVerboseLogging,
+  enableVerboseLogging,
+  logger
+} from "../../logger";
+import {
+  validateDefinition,
+  validateRemoteSource,
+  validateTemplateSource
+} from "./generate";
+
+beforeAll(() => {
+  enableVerboseLogging();
+  jest.setTimeout(10000);
```

Comment:
Is there a reason for setting a specific timeout?

Reply:
Since in my test, I actually clone a repo, It requires an increase in timeout by 5 seconds